docs/configuration-setup.rst

```diff
@@ -0,0 +1,193 @@
+when they give you an allocation of hours on the HPC system.
+
+.. code-block:: yaml
+
+   project: XYZW-1234
+
+If this sounds foreign to you or if you don't use project codes then you should
+leave this blank, or ask your IT administrator.
+
+
+Local Storage
+-------------
+
+When Dask workers run out of memory they typically start writing data to disk.
+This is often a wise choice on personal computers or analysis clusters, but can
+be unwise on HPC systems if they lack local storage.  When Dask workers try to
+write excess data to disk on systems that lack local storage this can cause the
+Dask process to die in unexpected ways.
+
+If your nodes have fast locally attached storage mounted somewhere then you
+should direct dask-jobqueue to use that location.
+
+.. code-block:: yaml
+
+   local-directory: /scratch
+
+Sometimes your job scheduler will give this location to you as an environment
+variable.  If so you should include that environment variable, prepended with
+the ``$`` sign and it will be expanded appropriately after the jobs start.
+
+.. code-block:: yaml
+
+   local-directory: $LOCAL_STORAGE
+
+
+No Local Storage
+----------------
+
+If your nodes do not have locally attached storage then we recommend that you
+turn off Dask's policy to write excess data to disk.  This must be done in a
+configuration file and must be separate from the ``jobqueue`` configuration
+section (though it is fine to include it in the same file).
+
+.. code-block:: yaml
+
+   jobqueue:
+     pbs:
+       cores: 36
+       memory: 100GB
+       ...
+
+   distributed:
+     worker:
+       memory:
+         target: False    # Avoid spilling to disk
+         spill: False     # Avoid spilling to disk
+         pause: .80       # Pause worker threads at 80% use
+         terminate: 0.95  # Restart workers at 95% use
+
+
+Network Interface
+-----------------
+
+HPC systems often have advanced networking hardware like Infiniband.
+Dask workers can take use of this network using TCP-over-Infiniband, this can
+yield improved bandwidth during data transfers.  To get this increased speed
+you often have to specify the network interface of your accelerated hardware.
+If you have sufficient permissions then you can find a list of all network
+interfaces using the ``ifconfig`` UNIX command
+
+.. code-block:: bash
+
+   $ ifconfig
+   lo          Link encap:Local Loopback                       # Localhost
+               inet addr:127.0.0.1  Mask:255.0.0.0
+               inet6 addr: ::1/128 Scope:Host
+   eth0        Link encap:Ethernet  HWaddr XX:XX:XX:XX:XX:XX   # Ethernet
+               inet addr:192.168.0.101
+               ...
+   ib0         Link encap:Infiniband                           # Fast InfiniBand
+               inet addr:172.42.0.101
+
+Alternatively, your IT administrators will have this information.
+
+
+Managing Configuration files
+----------------------------
+
+By default dask-jobqueue places a file at ``~/.config/dask/jobqueue.yaml`` with
```

Comment:
Should we add 'after first import with python' ? To avoid people looking for such a file just after conda or pip installing...